### DISCUSSION TITLE
Horizontal Scrolling <Container> or <Row>

### DISCUSSION THREAD

**Discussion by highjeans** [heroui-inc/heroui#658] [Q&A]
The title says it all. I can't seem to figure out how to do it. Is there any possible way to do it? Even better, can it be a snap scroll?

**Comment by tianenpang** [comment]
Hi @highjeans Here's an example [codesandbox.io/s/nextui-horizontal-scrolling-540pc9](https://codesandbox.io/s/nextui-horizontal-scrolling-540pc9?file=/pages/index.tsx).

**Reply by highjeans** [reply]
Thanks for taking the time out to show this! Just curious, what are the m and p css attributes used for?

**Comment by tianenpang** [comment]
The `m` and `p` are utilities, meaning shorthand for `margin` and `padding`, see more utilities in the docs: https://nextui.org/docs/theme/utilities

The `css` prop allow us to override the styles of components, for more details on overriding styles please check here: https://nextui.org/docs/theme/override-styles

**Reply by highjeans** [reply]
Thanks! I will let you know if it works with what I have.

**Reply by highjeans** [reply]
Another question, what is the flex attribute? My IDE does not recognize it.

**Reply by tianenpang** [reply]
https://developer.mozilla.org/en-US/docs/Web/CSS/flex

**Reply by highjeans** [reply]
lol autocorrect put flex instead of dflex. I was asking about dflex not flex.

**Reply by tianenpang** [reply]
`dflex` is a shorthand for `display, alignItems, justifyContent`, you can find it in the [utilities doc](https://nextui.org/docs/theme/utilities#utilities-1).

**Reply by highjeans** [reply]
Thanks a lot!
